src/main/webapp/script.js

```diff
@@ -0,0 +1,37 @@
+function countUnits(){
+  const inputElems = document.getElementsByName("search-units");
+  const arr = []
+  for (let i=0; i<inputElems.length; i++) {       
+    if (inputElems[i].checked){
+      arr.push(inputElems[i].value);
+    }
+  }
+  return arr;
+}
+
+function showClasses() {
+  const classes = document.getElementById("search-results");
+  const className = document.getElementById("search-class").value;
+  const profName = document.getElementById("search-prof").value;
+  const units = countUnits();
```

Comment:
Nice job splitting up this functionality.

Reply:
Thank you! :)